docs/_private_integrations/share-your-private-integration.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Share your private integration to users
+order: 3
+layout: post-toc
+redirect_from: /private_integrations/
+---
+# Share your private integration to users
+
+After you’ve built your private integration, you can invite users to start using your integration in their Zaps. There are two ways you can share your private integration with users:
+
+## Invite users with a public link
+
+You can share a link with users to all versions of your integration. 
+
+> Note:
+> * You can’t view invited users via a public link.
+> * You can’t revoke a link invite, once shared. If you require control over who accesses your private integration, we recommend inviting users by email. 
+
+
+<ol> 
```

Comment:
@gregilo because when I used links in lists, the link didn't work. So I used HTML instead to get around that. If I don't have to use HTML going forward, happy to do so.

Reply:
@jessveegee ahh, I see. I have an all-markdown suggestion that is properly rendering - I'll post it in a new thread and resolve this conversation.